.github/workflows/snapshot-releases.yml

```diff
@@ -7,10 +7,13 @@ concurrency:
   group: ${{ github.ref }}-${{ github.workflow }}
   cancel-in-progress: true
 on:
+  schedule:
+    #   Run on end of the day
+    - cron: '0 0 * * *'
   push:
     paths-ignore:
       - 'docs/**'
-    branches: [ main, v1, v2, v3, next ]
+    branches: [ fabric8-next-version ]
```

Comment:
I think we should keep releasing snapshots for `main` as well. Ideally, on every merge, even, which would mean splitting this out. Don't know. A snapshot a day is probably enough.

Reply:
AFAIK it works in a way that, this is not merged, so the github action will for this branch will take into account only the code on this branch. So since this PR never will be merged this should be enough to run the releases daily for this branch, but will work also for main (since that run  uses the code from there)